src/services/token.database.service.js

```diff
@@ -0,0 +1,15 @@
+const jwt = require("jsonwebtoken");
+
+module.exports.userTokenGenerator = async (existingUser) => {
+  const accessToken = jwt.sign(
+    {
+      existingUser: {
```

Comment:
wrong. study about JWT more. JWT has specific set of fields such as `sub`, `iss`, `exp`, `aud`